Makefile

```diff
@@ -1,5 +1,5 @@
 CORE_CONTAINERS := base nginx nginx2 php-fpm php-fpm2 jre-8 backupninja
-CHILD_CONTAINERS := phpunit golang rabbitmq wordpress elasticsearch elasticsearch-shared-data jenkins-slave jenkins kibana nginx-router wkhtmlpdf nginx-redirect casperjs  mongodb elasticsearch5
+CHILD_CONTAINERS := phpunit golang rabbitmq wordpress elasticsearch elasticsearch-shared-data jenkins-slave jenkins jenkins2 kibana nginx-router wkhtmlpdf nginx-redirect casperjs  mongodb elasticsearch5
```

Comment:
jenkins can be removed from the image list now as we don't need to update the latest build